be/src/common/config.h

```diff
@@ -718,6 +718,8 @@ CONF_Bool(enable_orc_late_materialization, "true");
 CONF_Int32(orc_file_cache_max_size, "8388608");
 CONF_Int32(orc_natural_read_size, "8388608");
 CONF_mBool(orc_coalesce_read_enable, "true");
+CONF_mBool(enable_orc_load_prefetch, "true");
```

Comment:
if you set true as default value, better to test against with TPCH/TPCDS query, or our scan benchmark.